### PR TITLE
Fix a few issues in standard.css

### PR DESCRIPTION
**Pull request by yarusome**
This PR fixes a few visible issues in the style sheet. Please see comments below for explanations.

### CODE REVIEW

File: resources.whatwg.org/standard.css [hunk 1]
```diff
@@ -304,6 +313,9 @@ dl.domintro dd p:last-child {
 .example::before {
   background: #222222;
 }
+td > .example, td > .note:not(span), td > .warning:not(span) {
+  margin-left: 0.8em;
+}
 td > .example:only-child {
   margin: 0 0 0 0.1em;
   padding: 0;
```
Comment (yarusome) [inline resources.whatwg.org/standard.css:321]: This fixes [13.2.2 Parse errors](https://html.spec.whatwg.org/multipage/parsing.html#parse-error-invalid-first-character-of-tag-name), where `.example::before` and `.note::before` intrude into adjacent cells.

**Comment by annevk** [comment]
Thank you, these improvements seem great. @domenic do you want to have a look as well?

**Comment by domenic** [comment]
Maybe add comments in the CSS, in addition to notes in a self-review of the PR?

**Comment by yarusome** [comment]
@domenic I just added some comments. What do you think?

**Review by domenic — APPROVED** [review]
Thanks!

**Pull request merged**